client/my-sites/site-settings/speed-up-site-settings.jsx

```diff
@@ -41,28 +81,59 @@ class SpeedUpSiteSettings extends Component {
 			isSavingSettings,
 			translate,
 			jetpackVersionSupportsLazyImages,
+			siteAcceleratorSupported,
+			photonModuleActive,
+			assetCdnModuleActive,
+			togglingSiteAccelerator,
 		} = this.props;
 		const isRequestingOrSaving = isRequestingSettings || isSavingSettings;
+		const cdnStatus = photonModuleActive || assetCdnModuleActive;
 
 		return (
 			<div className="site-settings__module-settings site-settings__speed-up-site-settings">
 				<Card>
 					<FormFieldset className="site-settings__formfieldset">
 						<SupportInfo
-							text={ translate( 'Hosts your image files on the global WordPress.com servers.' ) }
-							link="https://jetpack.com/support/photon/"
-						/>
-						<JetpackModuleToggle
-							siteId={ selectedSiteId }
-							moduleSlug="photon"
-							label={ translate( 'Serve images from our servers' ) }
-							description={ translate(
-								'Jetpack will optimize your images and serve them from the server ' +
-									'location nearest to your visitors. Using our global content delivery ' +
-									'network will boost the loading speed of your site.'
+							text={ translate(
+								"Jetpack's global Content Delivery Network (CDN) optimizes " +
+									'files and images so your visitors enjoy ' +
+									'the fastest experience regardless of device or location.'
 							) }
-							disabled={ isRequestingOrSaving || photonModuleUnavailable }
+							link="http://jetpack.com/support/site-accelerator/"
 						/>
+						<FormSettingExplanation className="site-settings__feature-description">
```

Comment:
Are we getting a little inconsistent when using `<FormSettingExplanation />` for the description of a card? For example, see the "Ads" or "Jetpack Search" cards under "Traffic" settings.

Reply:
Yes, probably a regular div. But it really depends on what's the style we want to display it with, and that's something that folks from @Automattic/jetpack-design would know.